app/controllers/api/v3/timelines/for_you_controller.rb

```diff
@@ -53,12 +56,12 @@ def set_for_you_feed
   end
 
   # Check account_from_acct finds an account
-  # Check the For You Beta Personal List
+  # Check AccountRelay that they are a Mammoth 2.0 User
   # @return [Boolean]
   def validate_mammoth_account
     return false if @account.nil?
 
-    PersonalForYou.new.personalized_mammoth_user?(acct_param)
+    PersonalForYou.new.mammoth_user?(acct_param)
   end
```

Comment:
we don't want to check for only personalized accounts, but rather any mammoth

Reply:
```rb
 def personalized_mammoth_user?(acct)
    user(acct).dig(:for_you_settings, :type) == 'personal'
  end
```
this makes a GET call for the user's profile

Reply:
when the user's profile is called directly from Moth.Social, if the acct doesn't exist we create it, if it's not marked 'local' we do that also. 🤦